File: Java/백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int[] arr = new int[3];
        int total = 0;
        
        for (int i = 0; i <3; i++) {
            arr[i] = sc.nextInt();
            
            total += arr[i];
        }
        
        if (total == 180 && (arr[0] == arr[1] && arr[1] == arr[2]))
            System.out.print("Equilateral");
        else if (total == 180 && (arr[0] == arr[1] || arr[1] == arr[2] || arr[2] == arr[0]))
            System.out.print("Isosceles");
        else if (total == 180 && (arr[0] != arr[1] && arr[1] != arr[2] && arr[2] != arr[0]))
            System.out.print("Scalene");
        else
            System.out.print("Error");
        
        sc.close();
    }
}

File: C++17/백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cc
#include <iostream>
using namespace std;

int main () {
    int arr[3];
    int total = 0;
    
    for (int i = 0; i < 3; i++) {
        cin >> arr[i];
        
        total += arr[i];
    }
    
    if (total == 180 && (arr[0] == arr[1] && arr[1] == arr[2]))
        cout << "Equilateral";
    else if (total == 180 && (arr[0] == arr[1] || arr[1] == arr[2] || arr[2] == arr[0]))
        cout << "Isosceles";
    else if (total == 180 &&(arr[0] != arr[1] && arr[1] != arr[2] && arr[2] != arr[0]))
        cout << "Scalene";
    else
        cout << "Error";
    
    return 0;
}